client/client.go

```diff
@@ -427,6 +427,19 @@ func (client *OpenFgaClient) getAuthorizationModelId(authorizationModelId *strin
 	return modelId, nil
 }
 
+// helper function to validate the connection (i.e., get token)
+func (client *OpenFgaClient) validateAuthorizationModel(ctx _context.Context, authorizationModelId *string) error {
```

Comment:
Can you call this checkApiConnection? The intention is to make sure we can talk to the API, not read the model